OpenRA.Game/Sound/Sound.cs

```diff
@@ -86,8 +86,8 @@ public static SoundDevice[] AvailableDevices()
 		{
 			var defaultDevices = new[]
 			{
-				new SoundDevice("AL", null, "Default Output"),
-				new SoundDevice("Null", null, "Output Disabled")
+				new SoundDevice("AL", null, FieldLoader.Translate("SOUND-DEFAULT")),
```

Comment:
As I mentioned in IRC before you did any of these changes - FieldLoader has nothing conceptually to do with translations (except that it uses them), so it is not an appropriate class to do the translations.  This should be done via ModData instead.